### DISCUSSION TITLE
multiple uibuilder.onChange()

### DISCUSSION THREAD

**Discussion by taufiksu** [TotallyInformation/node-red-contrib-uibuilder#113] [Q&A] [answered]
Is that possible to have multiple uibuilder.onChange() for receiving message?

**Comment by TotallyInformation** [comment, marked as answer]
Hi, thanks for engaging in the discussion. 👍 

Yes, you can have as many as you like. They will _all_ be executed when a message is received.

**Reply by taufiksu** [reply]
Hi, thanks for reply, in my case i have some page that init multiple time when the web page is opened, then inside the init is the uibuilder code onChange('msgsReceived'). Based my experience because the page will re init everytime open, then the onChange('msgsReceived') behave like multiple time init also. This cause when i received message will have more than one time. Maybe i did mistake when using it. I will check again my code.
